src/pcapi/validation/routes/offers.py

```diff
@@ -6,6 +7,14 @@ def check_offer_type_is_valid(offer_type_name):
     if not ProductType.is_thing(offer_type_name) and not ProductType.is_event(offer_type_name):
         api_error = ApiErrors()
         api_error.add_error("type", "Le type de cette offre est inconnu")
+        api_error.add_error("subcategory", "La sous-catégorie de cette offre est inconnue")
+        raise api_error
+
+
+def check_offer_subcategory_is_valid(offer_subcategory_id):
+    if not offer_subcategory_id in ALL_SUBCATEGORIES_DICT.keys():
```

Comment:
(détail / style)

```suggestion
    if offer_subcategory_id not in ALL_SUBCATEGORIES_DICT:
```

plus idiomatique